package.json

```diff
@@ -24,7 +25,10 @@
   },
   "homepage": "https://github.com/warehouseai/release-line#readme",
   "dependencies": {
-    "datastar": "^1.7.0"
+    "aws-liveness": "^1.1.0",
+    "aws-sdk": "^2.563.0",
+    "dynamodb-x": "^1.2.3",
```

Comment:
should this be a dev-dep?

Reply:
also aws-liveness & aws-sdk?

Reply:
yea devDep those `--save-dev`

Reply:
oops, must have been when I did --saveDev instead of --save-dev without realizing :sweat_smile: